virttest/libvirt_xml/devices/disk.py

```diff
@@ -231,6 +233,17 @@ def new_backingstore(self, **dargs):
             setattr(new_one, key, value)
         return new_one
 
+    def get_backingstore_list(self):
+        """
+        Return a disk backingstore list
+        """
```

Comment:
The docstring could be updated for :return:

Reply:
@dzhengfy 
updated